otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -20,16 +20,53 @@ exporters:
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudpubsubexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlemanagedprometheusexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/humioexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/instanaexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerthrifthttpexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logzioexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/pulsarexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sentryexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/skywalkingexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/splunkhecexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumologicexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/tanzuobservabilityexporter v0.62.0"
+  # - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/tencentcloudlogserviceexporter v0.62.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter v0.62.0"
```

Comment:
@SumoLogic/open-source-collection-team

Are we satisfied with enabled components?

I would left commented exporters to easier spot new/removed exporters in the future

Reply:
We decided to restrict exporters as much as possible, so adding only `prometheusexporter`